src/control/cmd/daos_server/storage_legacy.go

```diff
@@ -1,5 +1,5 @@
 //
-// (C) Copyright 2022-2023 Intel Corporation.
```

Comment:
Should be removed ?

Reply:
Right, this was git-hook automated unfortunately. I will correct if I have to re- push.